test/test_utils.erl

```diff
@@ -0,0 +1,32 @@
+-module(test_utils).
```

Comment:
This module should be called `hank_test_utils`. There are too many `test_utils` around already.